mlrun/config.py

```diff
@@ -523,11 +523,11 @@
         "parquet_batching_max_events": 10_000,
         "parquet_batching_timeout_secs": timedelta(minutes=1).total_seconds(),
         # See mlrun.model_monitoring.db.stores.ObjectStoreFactory for available options
-        "store_type": "v3io-nosql",
```

Comment:
following our discussion with Saar, maybe leave it for now and add a deprecated note as you did for the `stream` value, at least until we can confirm that it doesn't break anything such as installing CE